crates/apub/src/activities/create_or_update/post.rs

```diff
@@ -187,6 +191,22 @@ impl ActivityHandler for CreateOrUpdatePage {
       score: 1,
     };
     PostLike::like(context.pool(), &like_form).await?;
+
+    // Calculate initial hot_rank for post
+    let conn = &mut get_conn(context.pool()).await?;
+
+    diesel::update(post_aggregates::table)
```

Comment:
Same as above, add this as a helper function to `crates/db_schema/src/impls/post.rs` or `crates/db_schema/src/impls/post_aggregates.rs`